L/libgmime/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libgmime"
+version = v"3.2.15"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/jstedfast/gmime/releases/download/$(version)/gmime-$(version).tar.xz", "84cd2a481a27970ec39b5c95f72db026722904a2ccf3fdbd57b280cf2d02b5c4")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd gmime-3.2.15/
+echo '#define ICONV_ISO_INT_FORMAT "iso-%u-%u"' > iconv-detect.h
+echo '#define ICONV_ISO_STR_FORMAT "iso-%u-%s"' >> iconv-detect.h
+echo '#define ICONV_SHIFT_JIS "shift-jis"' >> iconv-detect.h
+echo '#define ICONV_10646 "UCS-4BE"' >> iconv-detect.h
+
+export LDFLAGS="$LDFLAGS -liconv"
```

Comment:
Why is this needed?
```suggestion
export LDFLAGS="-liconv"
```

Reply:
Please answer the question

Reply:
same as above

Reply:
Without adding iconv-detect.h there is an error:
```
gmime-charset.c:49:10: fatal error: iconv-detect.h: No such file or directory
 #include "iconv-detect.h"
          ^~~~~~~~~~~~~~~~
compilation terminated.
make[3]: *** [Makefile:917: gmime-charset.lo] Error 1
make[3]: *** Waiting for unfinished jobs....
```

Reply:
Without `ac_cv_have_iconv_detect_h=yes` key
```
checking preferred charset formats for system iconv... cannot run test program while cross compiling
configure: error: Copy target system's iconv-detect.h to . and configure with ac_cv_have_iconv_detect_h=yes
```